src/game/server/NextBot/Player/NextBotPlayer.h

```diff
@@ -411,6 +429,22 @@ inline void NextBotPlayer< PlayerType >::ReleaseJumpButton( void )
 	m_jumpButtonTimer.Invalidate();
 }
 
+#ifdef NEO
+template < typename PlayerType >
+inline void NextBotPlayer< PlayerType >::PressThermopticButton( float duration )
+{
+	m_inputButtons |= IN_THERMOPTIC;
+	m_thermopticButtonTimer.Start( duration );
+}
+
+template < typename PlayerType >
+inline void NextBotPlayer< PlayerType >::ReleaseThermopticButton( void )
+{
+	m_inputButtons &= ~IN_THERMOPTIC;
+	m_thermopticButtonTimer.Invalidate();
+}
+#endif
+
```

Comment:
Ideally, all of these neo-specific methods should live in `class CNEOBot : public NextBotPlayer< CNEO_Player >` instead of being ifdef'd here inline with the base class. But I'm ok leaving this here if you don't feel like refactoring it as part of this PR.